python/RDS_ENHANCED_MONITORING_ENABLED/RDS_ENHANCED_MONITORING_ENABLED_test.py

```diff
@@ -0,0 +1,201 @@
+import sys
+import unittest
+try:
+    from unittest.mock import MagicMock
+except ImportError:
+    from mock import MagicMock
+import json
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::RDS::DBInstance'
+
+#############
+# Main Code #
+#############
+
+CONFIG_CLIENT_MOCK = MagicMock()
+STS_CLIENT_MOCK = MagicMock()
+
+class Boto3Mock():
+    @staticmethod
+    def client(client_name, *args, **kwargs):
+        if client_name == 'config':
+            return CONFIG_CLIENT_MOCK
+        if client_name == 'sts':
+            return STS_CLIENT_MOCK
+        raise Exception("Attempting to create an unknown client")
+
+sys.modules['boto3'] = Boto3Mock()
+
+RULE = __import__('RDS_ENHANCED_MONITORING_ENABLED')
+
+class ComplianceTest(unittest.TestCase):
+    rule_invalid_parameter = '{"monitoringInterval":"12"}'
+    rule_valid_parameter = '{"monitoringInterval":"5"}'
+    rule_parameter_mismatch = '{"monitoringInterval":"10"}'
+
+    valid_em_interval_configured = {
+        "functionName": "test_function",
```

Comment:
I do not see use of functionName & functionArn in configuration item, we just need 'monitoringInterval'.
Kindly remove the others